data/json/items/melee/spears_and_polearms.json

```diff
@@ -601,6 +601,27 @@
     "qualities": [ [ "COOK", 1 ] ],
     "flags": [ "SPEAR", "REACH_ATTACK", "DURABLE_MELEE" ]
   },
+  {
+    "id": "pollaxe",
+    "type": "GENERIC",
+    "symbol": "/",
+    "color": "light_gray",
+    "name": { "str": "pollaxe", "str_pl": "pollaxes" },
```

Comment:
```suggestion
    "name": { "str": "pollaxe" },
```

Reply:
The weapon is named POLEAXE not pollaxe (and it's easy to know because, as the description rightly states, it's a polearm with an axe head)

Reply:
well, no, not really. 
(https://en.wikipedia.org/wiki/Pollaxe)
it can be called a poleaxe, but after that point why not just call it a polax?
also, sorry for the late reply, i was in work.

Reply:
Even the wikipedia article you linked says at the top: Several variant spellings redirect here... (among them, "poleaxe")
I was NOT familiar with the pollaxe spelling, but "poleaxe" is a very common weapon in various pen and paper and computer rpg games - therefore I would lean towards the latter as better established.
And due to variant spellings, definitely needs a translator comment.

Reply:
not to be offensive or petty anything, but...

(Weapons such as the halberd, bardiche, and Danish axe are sometimes mistakenly called pollaxes as they are indeed axes mounted on poles, but many etymological authorities consider the poll- prefix historically unrelated to "pole", instead meaning "head".)

wikipedia says this and then cites some good sources, like the oxford dictionary, so i think pollaxe might be the correct spelling.

Reply:
not to be offensive or petty anything, but...

(Weapons such as the halberd, bardiche, and Danish axe are sometimes mistakenly called pollaxes as they are indeed axes mounted on poles, but many etymological authorities consider the poll- prefix historically unrelated to "pole", instead meaning "head".)

wikipedia says this and then cites some good sources, like the oxford dictionary, so i think pollaxe might be the correct spelling.

Reply:
The oxford dictionary as cited says, "In the 16th c. the word began to be written by some pole-axe (which after 1625 became the usual spelling)"
So, the only opinion stated in the article is that it's poleaxe.  Pollaxe is revisionist, we use modern spellings, not archaic ones.

Reply:
dont agree, but will do anyways.